<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rofiapp_MainWindow</class>
 <widget class="QMainWindow" name="Rofiapp_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>763</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RoFI User Application</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rofiapp_resources.qrc">
    <normaloff>:/rofiapp/rofi-icona.png</normaloff>:/rofiapp/rofi-icona.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_conf">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Configuration Editor</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QSplitter" name="splitter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="CompatQVTKWidget" name="qvtkWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>2</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
            <widget class="QWidget" name="horizontalLayoutWidget">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPlainTextEdit" name="configTextWindow">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="plainText">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="connectedLabel">
                  <property name="visible">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Connected</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QCheckBox" name="connectedCheckBox">
                  <property name="visible">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="angleAlphaLabel">
                  <property name="visible">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Angle α</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QDial" name="angleAlphaDial">
                  <property name="visible">
                   <bool>false</bool>
                  </property>
                  <property name="minimum">
                   <number>-90</number>
                  </property>
                  <property name="maximum">
                   <number>90</number>
                  </property>
                  <property name="notchesVisible">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="angleBetaLabel">
                  <property name="visible">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Angle β</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QDial" name="angleBetaDial">
                  <property name="visible">
                   <bool>false</bool>
                  </property>
                  <property name="minimum">
                   <number>-90</number>
                  </property>
                  <property name="maximum">
                   <number>90</number>
                  </property>
                  <property name="notchesVisible">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="angleGammaLabel">
                  <property name="visible">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Angle ɣ</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QDial" name="angleGammaDial">
                  <property name="visible">
                   <bool>false</bool>
                  </property>
                  <property name="minimum">
                   <number>-180</number>
                  </property>
                  <property name="maximum">
                   <number>180</number>
                  </property>
                  <property name="wrapping">
                   <bool>true</bool>
                  </property>
                  <property name="notchesVisible">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_about">
       <attribute name="title">
        <string>About</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="aboutLabel">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="rofiapp_resources.qrc">:/rofiapp/rofi-logo-2row-black.svg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="aboutLabel2">
            <property name="text">
             <string>http://ro.fi.muni.cz</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionToggle_full_screen"/>
    <addaction name="actionChange_background"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionShow_sphere"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadConf"/>
   <addaction name="actionSaveConf"/>
   <addaction name="actionShowConf"/>
   <addaction name="actionResetCamera"/>
   <addaction name="actionExportConf"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionToggle_full_screen">
   <property name="text">
    <string>Toggle full screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionChange_background">
   <property name="text">
    <string>Change background</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionShow_sphere">
   <property name="text">
    <string>Show sphere</string>
   </property>
  </action>
  <action name="actionLoadConf">
   <property name="text">
    <string>Load conf</string>
   </property>
  </action>
  <action name="actionSaveConf">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save conf</string>
   </property>
  </action>
  <action name="actionShowConf">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </action>
  <action name="actionResetCamera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reset Camera</string>
   </property>
  </action>
  <action name="actionExportConf">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export as bitmap</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CompatQVTKWidget</class>
   <extends>QWidget</extends>
   <header>CompatQVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rofiapp_resources.qrc"/>
 </resources>
 <connections/>
</ui>
